sale_expected_delivery_date/model/sale.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Kitti U. <kittiu@gmail.com>
+#    Copyright 2015 Ecosoft
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from openerp import models, fields, api
+from datetime import datetime
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    date_expected = fields.Date(
+        string='Expected Delivery Date',
+        required=True,
```

Comment:
You should provide a post-init hook to initialize all existing orders to a proper value, because if not, all will be set at the day of the module installation.